### DISCUSSION TITLE
AWS S3 guide

### DISCUSSION THREAD

**Discussion by samg666** [Budibase/budibase#4869] [Help]
Hello,

May I know if there's any guide on how to use AWS S3 on Budibase, or is there any template apps included AWS S3? Tried search but couldn't find any documents yet. The documents better with both Budibase and AWS S3 setting.

**Comment by mjashanks** [comment]
Sorry Sam, working on Documentation. We did have an issue with S3 connector, which has been resolved in the latest release `1.0.86`

**Comment by murilobd** [comment]
Hi, talking about S3 connector, would be possible to explain how to replace MinIO for S3 using budibase-cli/docker installation?
I found this issue (https://github.com/Budibase/budibase/discussions/5602) that explains how to do it with k8s, but didn't manage to adapt it to docker
